### PR TITLE
Exercise_12 Version5

### PR DESCRIPTION
**Pull request by serine-galstyan**

### CODE REVIEW

File: serine.galstyan/repo/Lesson_3/exercise_12/exercise_12.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/evn python3
+
+liter_or_less = int(input("Enter the number of containers that holding 1 liter or less:"))
```
Comment (Zohrabya) [inline serine.galstyan/repo/Lesson_3/exercise_12/exercise_12.py]: CSW: Space after the user message.

File: serine.galstyan/repo/Lesson_3/exercise_12/exercise_12.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+more_than_liter = int(input("Enter the number of containers that holding more than 1 liter:"))
+total = (0.10 * liter_or_less) + (0.25 * more_than_liter)
```
Comment (Zohrabya) [inline serine.galstyan/repo/Lesson_3/exercise_12/exercise_12.py]: LW: What represents numbers 0.10 and 0.25. Use constant variables for these numbers.

File: serine.galstyan/repo/Lesson_3/exercise_12/exercise_12.py [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+final = round(total, 2)
```
Comment (Zohrabya) [inline serine.galstyan/repo/Lesson_3/exercise_12/exercise_12.py]: LW: Unnecessary line.

**Review by Zohrabya — COMMENTED** [review]
exercise_12: 47 - reply

File: serine.galstyan/repo/Lesson_3/exercise_12/exercise_12.py [hunk 4]
```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/evn python3
```
Comment (Zohrabya) [inline serine.galstyan/repo/Lesson_3/exercise_12/exercise_12.py]: CSW: Wrong shebang line

File: serine.galstyan/repo/Lesson_3/exercise_12/exercise_12.py [hunk 5]
```diff
@@ -0,0 +1,8 @@
...
+
+liter_or_less = int(input("Enter the number of containers that holding 1 liter or less: "))
+more_than_liter = int(input("Enter the number of containers that holding more than 1 liter: "))
+lightweight_container_deposit = 0.10
```
Comment (Zohrabya) [inline serine.galstyan/repo/Lesson_3/exercise_12/exercise_12.py]: CSW: Constant value

**Review by Zohrabya — COMMENTED** [review]
exercise_12: 45 - reply

File: serine.galstyan/repo/Lesson_3/exercise_12/exercise_12.py (hunk 5 above)
Comment (serine-galstyan) [reply serine.galstyan/repo/Lesson_3/exercise_12/exercise_12.py]: lightweight_container_deposit and heavy_container_deposit variables represent 0.10 and 0.25 numbers, as it was indicated to do in the previous comment.

File: serine.galstyan/repo/Lesson_3/exercise_12/exercise_12.py [hunk 6]
```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env python3
+
+liter_or_less = int(input("Enter the number of containers that holding 1 liter or less: "))
+more_than_liter = int(input("Enter the number of containers that holding more than 1 liter: "))
+LIGHTWEIGHT_CONTAINER_DEPOSIT = 0.10
```
Comment (Zohrabya) [inline serine.galstyan/repo/Lesson_3/exercise_12/exercise_12.py]: CST: LESS_CONTAINER_DEPOSIT will be more proper

**Review by Zohrabya — COMMENTED** [review]
exercise_12: 45 -reply

**Review by Zohrabya — APPROVED** [review]
exercise_12: 95 - Accepted

**Pull request merged**